doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -356,8 +356,9 @@ KRKNWK-17718: Failure at TC-DGSW-1.1 Matter Certification test
   The issue happens due to a wrong Software Diagnostic cluster feature map with features enabled by default.
 
   **Workaround:** Set the default value of the Software Diagnostic cluster feature map to 0 using the ZAP Tool and regenerate files.
+  and manually cherry-pick and apply the commit with the fix to ``sdk-connectedhomeip`` (commit hash: ``70dd449ff943159365466ad5125f42a5bdcbfc0b``)
 
-.. rst-class:: v2-4-2 v2-4-1 v2-4-0
+.. rst-class:: v2-4-2 v2-4-1 v2-4-0 v2-3-0
```

Comment:
Is this an intended change? This line affects issue KRKNWK-17699, not KRKNWK-17718.